feature-libs/organization/administration/core/store/effects/org-unit.effect.ts

```diff
@@ -127,7 +129,11 @@ export class OrgUnitEffects {
       this.orgUnitConnector
         .update(payload.userId, payload.unitCode, payload.unit)
         .pipe(
-          switchMap(() => [new OrganizationActions.OrganizationClearData()]),
+          switchMap((_data) => [
```

Comment:
we can drop `_data` here.

Reply:
I can drop if it's really necessary, but we left that cases as it will be used after fix workaround